flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesUtilsTest.java

```diff
@@ -87,41 +88,43 @@ public void testGetJobManagerStartCommand() {
 		final Configuration cfg = new Configuration();
 
 		final String jmJvmOpts = "-DjmJvm";
+		final int jmMemoryAfterCutoff = BootstrapTools.calculateHeapSize(jobManagerMem, cfg);
+		final String jmJvmMem = String.format("-Xms%dm -Xmx%dm", jmMemoryAfterCutoff, jmMemoryAfterCutoff);
```

Comment:
We can still make these two vars as class level, and then `jmJvmMem` is not needed to pass into below `getJobManagerExpectedCommand` every time, because it is actually never changed.

Reply:
Make sense. I will try to simply the codes.